packages/api/lambdas/process-s3-dead-letter-archive.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const log = require('@cumulus/common/log');
+
+const { getJsonS3Object, listS3ObjectsV2, deleteS3Object } = require('@cumulus/aws-client/S3');
+const { getKnexClient } = require('@cumulus/db');
+const { getMessageExecutionName } = require('@cumulus/message/Executions');
+
+const { writeRecords } = require('./sf-event-sqs-to-db-records');
+
+async function processDeadLetterArchive({
+  knex,
+  bucket = process.env.system_bucket,
+  path = `${process.env.stackName}/dead-letter-archive/sqs/`,
+  writeRecordsFunction = writeRecords,
+}) {
+  // TODO: figure out a way to cap this so we don't read a billion objects into memory
+  const deadLetterObjects = await listS3ObjectsV2({
+    Bucket: bucket,
+    Prefix: path,
+  });
```

Comment:
Happy to take suggestions here.

Reply:
Resolved in https://github.com/nasa/cumulus/pull/2181/commits/54544cb48d8937dae00a79301bd382354964a4f0